src/main/Constants/app-constant.ts

```diff
@@ -1,10 +1,58 @@
 export const AppConstants = {
-
   Morning: 'Morning',
   Afternoon: 'Afternoon',
   Evening: 'Evening',
   Today: 'Today',
   Yesterday: 'Yesterday',
   Tomorrow: 'Tomorrow',
   Alphabetical: 'Alphabetical',
+  Times: 'Times',
+  Daily: 'Daily',
+  Per_Day: 'Per Day',
+  habits_Order: 'My Habits Order',
+  repeat: 'Repeat Every 2',
+  months: 'Every month on the 1th',
```

Comment:
I think it should be "Every month on the 1st"

Reply:
On Habitify App its 1th thats why I named it as 1th

Reply:
I think it is ok to make it 1st